### PR TITLE
[zh-cn]: update the translation of Atomics.isLockFree()

### PR DESCRIPTION
**Pull request by JasonLamv-t**
### Description

Same as the title.

### Motivation

Ditto.

### Additional details

None.

### Related issues and pull requests

#17175

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/atomics/islockfree/index.md [hunk 1]
```diff
@@ -1,31 +1,35 @@
...
 ### 参数
 
 - `size`
-  - : 整形。字节长度，通常为[`TypedArray.BYTES_PER_ELEMENT`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/BYTES_PER_ELEMENT)
+  - : 要检查的大小（以字节为单位）。
 
 ### 返回值
 
-{{jsxref("Boolean")}} 是否为能够使用原子操作的[`TypedArray`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/BYTES_PER_ELEMENT)的标准字节长度之一。
+指示操作是否不涉及锁的 `true` 或 `false`。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/atomics/islockfree/index.md:27]:
```suggestion
`true` 或 `false`，指示操作字节是否不涉及锁。
```

File: files/zh-cn/web/javascript/reference/global_objects/atomics/islockfree/index.md [hunk 2]
```diff
@@ -1,31 +1,35 @@
...
 slug: Web/JavaScript/Reference/Global_Objects/Atomics/isLockFree
+l10n:
+  sourceCommit: 3f0cd840cd9575701c65b8c6a1e172a2b0c3bd62
 ---
 
-{{JSRef}} {{SeeCompatTable}}
+{{JSRef}}
 
-静态方法 **`Atomics.isLockFree()`** 用于校验是否能够使用原子操作的 [`TypedArray`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/BYTES_PER_ELEMENT) 的标准字节长度之一。若该字节长度为可处理的 TypedArray 标准字节长度之一则返回 `true`。TypedArray 的标准字节长度参见 [BYTES_PER_ELEMENT](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/BYTES_PER_ELEMENT)
+**`Atomics.isLockFree()`** 静态方法用于判断 `Atomics` 在使用给定元素字节大小应用于类型化数组时是否会使用锁或原子硬件操作。如果给定的大小不是整数类型化数组类型的 [BYTES_PER_ELEMENT](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/BYTES_PER_ELEMENT) 属性之一，则返回 `false`。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/atomics/islockfree/index.md:10]:
```suggestion
**`Atomics.isLockFree()`** 静态方法用于判断 `Atomics` 在将具有给定字节大小的元素应用于类型化数组时是否会使用锁或原子硬件操作。如果给定的大小不是整数类型化数组类型的 [BYTES_PER_ELEMENT](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/BYTES_PER_ELEMENT) 属性之一，则返回 `false`。
```

**Review by yin1999 — COMMENTED** [review]
How about this?

**Comment by JasonLamv-t** [comment]
> How about this?

Much better. 👍

**Review by yin1999 — APPROVED** [review]
Thanks @JasonLamv-t

**Pull request merged**